modules/core/src/main/java/org/apache/ignite/internal/processors/bulkload/BulkLoadProcessor.java

```diff
@@ -120,7 +134,7 @@ public void onError(Exception failReason) {
         if (isClosed)
             return;
 
-        try {
+        try (MTC.TraceSurroundings ignored = MTC.supportContinual(qrySpan)) {
```

Comment:
Should we use support() instead of supportContinual() here in order to close span on bulkLoadProcessor.close()?